tests/mypy/outputs/1.1.1/mypy-plugin-strict-no-any_ini/dataclass_no_any.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import ConfigDict
+from pydantic.dataclasses import dataclass
+
+
+@dataclass
+# MYPY: error: Expression type contains "Any" (has type overloaded function)  [misc]
+class Foo:
+    foo: int
+
+
+@dataclass(config=ConfigDict(title='Bar Title'))
+# MYPY: error: Expression type contains "Any" (has type "Type[ConfigDict]")  [misc]
+# MYPY: error: Expression type contains "Any" (has type "ConfigDict")  [misc]
+class Bar:
+    bar: str
```

Comment:
@hramezani did you intentionally add this error?

Reply:
No, It was generated by the script. Actually, we can say by your change.

Reply:
The point of this test was specifically to make sure this error _didn't_ get created. However, I've discovered that using a type alias for the callable in `json_encoders` seems to make this go away. (At least with mypy 1.1.1...), which I've just pushed in a commit. So hopefully that resolves the issue.